Automattic/jetpack
# IDC: allow tags in customized texts

#### Changes proposed in this Pull Request:
- allows the `<em>...</em>` and `<strong>...</strong>` tags in customized texts

#### Jetpack product discussion
p9dueE-3Xf-p2#comment-7006

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Bring the site to the Identity Crisis mode.
2. Install and activate the new version of the [customizer plugin](https://gist.github.com/sergeymitr/f4257d64be6282e36b877a18a30d819f).
3. Go to the admin area and confirm that the customized content contains the `<em>...</em>` and `<strong>...</strong>` tags in it, rendered properly.
4. Log in as a non-admin user to see the non-admin IDC screen. Confirm that the text there also contains the `em` and `strong` tags, properly rendered.
5. Login as an admin again, and click "Move your settings". The screen should change to the "Successfully migrated" one, confirm that it also contains the `em` and `strong` tags, properly rendered.
